lib/bap_llvm/llvm_elf_loader.hpp

```diff
@@ -137,6 +138,10 @@ void section_header(const T &hdr, const std::string &name, uint64_t base, ogre_d
     bool w = static_cast<bool>(hdr.sh_flags & ELF::SHF_WRITE);
     bool x = static_cast<bool>(hdr.sh_flags & ELF::SHF_EXECINSTR);
     s.entry("section-flags") << name << w << x;
+    if (x)
+        s.entry("code-entry") << name << hdr.sh_offset << hdr.sh_size;
+    else
```

Comment:
again, `not SHF_EXECINSTR` does not imply that it is program data section, there are quite a few auxiliary and ancillary data structures used by the linker itself and various instrumentation tools, e.g., `.hash` section or `.dwarf`.